use_case_examples/hybrid_model/infer_hybrid_llm_generate.py

```diff
@@ -0,0 +1,86 @@
+"""Showcase for the hybrid model converter."""
+import json
+import time
+from pathlib import Path
+
+import torch
+from torch.backends import mps
+from transformers import AutoModelForCausalLM, AutoTokenizer, TextStreamer
+
+from concrete.ml.torch.hybrid_model import HybridFHEMode, HybridFHEModel
+
+if __name__ == "__main__":
+    # Use configuration dumped by compilation
+    with open("configuration.json", "r") as file:
+        configuration = json.load(file)
+    module_names = configuration["module_names"]
+
+    device = "cpu"
+    if torch.cuda.is_available():
+        device = "cuda"
+    if mps.is_available():
+        device = "mps"
+    print(f"Using device: {device}")
+
+    # Get GPT2 from Hugging Face
+    model_name = "gpt2"
+    # Avoid having / in the string
+    model_name_no_special_char = model_name.replace("/", "_")
+    tokenizer = AutoTokenizer.from_pretrained(model_name)
+    model = AutoModelForCausalLM.from_pretrained(
+        model_name,
+        device_map=device,
+        trust_remote_code=True,
+    )
+
+    # Modify model to use remote FHE server instead of local weights
+    hybrid_model = HybridFHEModel(
```

Comment:
Why are we rebuilding the hybrid model from scratch here instead of re-using the one computed in compile_hybrid_llm.py?

Reply:
How would you do that?
I had two solutions in mind:
- Just use `torch.load` but that uses pickle and I'm not a big fan of it
- Use the `.pth` file but that would have mean re-creating the HybridModel anyways.

Reply:
I think you can store the .pth along with the state dict and then when you will load the custom remote layer will already be in there. So you can then simply call model().

Reply:
You can't get the remote modules without some pickle

Reply:
Yes you can because you have them imported. When torch.load is called it will try to call the RemoteModule. If it's not imported then it will fail otherwise it will use the one we have defined.

Reply:
We can probably do this in another issue